base/record_replay.cc

```diff
@@ -582,12 +582,10 @@ int NewIdAnyThread(const char* name) {
 }
 
 void Crash(const char* format, ...) {
-  if (IsRecordingOrReplaying()) {
```

Comment:
I thought we agreed no more sneaky changes :)

Reply:
I'm not sure abuot "agreed" 🤣 👀 👀 
Having individual PRs for all these miniscule changes, and then getting reviews for them, checking tests etc etc etc etc seems cumbersome